packages/amplify-codegen/src/commands/models.js

```diff
@@ -79,10 +79,10 @@ async function generateModels(context) {
   const generatedCode = await Promise.all(codeGenPromises);
 
   // clean the output directory before re-generating models
-  const cleanOutputPath = FeatureFlags.getBoolean('codegen.cleanGeneratedModelsDirectory');
-  if (cleanOutputPath) {
-    await fs.emptyDir(outputPath);
```

Comment:
Is this the only place this command is run?

Reply:
Yes